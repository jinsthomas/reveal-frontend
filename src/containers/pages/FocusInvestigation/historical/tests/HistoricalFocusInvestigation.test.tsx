import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { createBrowserHistory } from 'history';
import React from 'react';
import { Router } from 'react-router';
import HistoricalFocusInvestigation from '../../historical';

const history = createBrowserHistory();

describe('containers/pages/HistoricalFocusInvestigation', () => {
  beforeEach(() => {
    jest.resetAllMocks();
  });

  it('renders without crashing', () => {
    shallow(
      <Router history={history}>
        <HistoricalFocusInvestigation />
      </Router>
    );
  });

  it('renders HistoricalFocusInvestigation correctly', () => {
    const wrapper = mount(
      <Router history={history}>
        <HistoricalFocusInvestigation />
      </Router>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
    wrapper.unmount();
  });
});
