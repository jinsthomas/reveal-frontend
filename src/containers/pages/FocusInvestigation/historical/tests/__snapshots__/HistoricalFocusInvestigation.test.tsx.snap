// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/pages/FocusInvestigation renders FocusInvestigation correctly 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <HistoricalFocusInvestigation>
    <div>
      <HeaderBreadcrumb>
        <div>
          <Breadcrumb
            aria-label="breadcrumb"
            className="reveal-breadcrumb"
            listTag="ol"
            tag="nav"
          >
            <nav
              aria-label="breadcrumb"
              className="reveal-breadcrumb"
            >
              <ol
                className="breadcrumb"
              >
                <BreadcrumbItem
                  tag="li"
                >
                  <li
                    className="breadcrumb-item"
                  >
                    <Link
                      replace={false}
                      to="/irs"
                    >
                      <a
                        href="/irs"
                        onClick={[Function]}
                      >
                        IRS
                      </a>
                    </Link>
                  </li>
                </BreadcrumbItem>
                <BreadcrumbItem
                  active={true}
                  tag="li"
                >
                  <li
                    aria-current="page"
                    className="active breadcrumb-item"
                  >
                    The Current Page
                  </li>
                </BreadcrumbItem>
              </ol>
            </nav>
          </Breadcrumb>
        </div>
      </HeaderBreadcrumb>
      <div>
        Focus Investigation
      </div>
    </div>
  </HistoricalFocusInvestigation>
</Router>
`;
